configure

```diff
@@ -2926,7 +2926,7 @@ $as_echo_n "checking for python interpreter for cross build... " >&6; }
 	    if $interp -c "import sys;sys.exit(not '.'.join(str(n) for n in sys.version_info[:2]) == '$PACKAGE_VERSION')"; then
 	        break
 	    fi
-            interp=
```

Comment:
Why this line is removed? Is ` configure` generated from ` configure.ac`? What version of autoconf you have used?

Reply:
I haven't checked in a regenerated configure yet. What version do you use?

Reply:
The last stable version is 2.69.